README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## This is my brief Biography!
+
+My name is Eleanor Anderson. I am a current fourth year undergraduate student majoring in Economics and Psychology. I am planning on graduating this quarter and am applying to graduate schools. I wanted to take this class becuase I have had a handful of classes that use R but I never truly learned how R works, I mostly learned how to plug and chug. I am **very** *excited* to learn how to actually use R!
+
+My main hobbies and interests are as follows:
+
+* Dance
+* Music (both playing and listening)
+* Cooking and Baking
+* Knitting 
+
+### Dance
+
+Dance is *probably* what I spend the majority of my time doing! It is so freeing and I get to spend time with some of the **greatest** people I know! I have attached a picture of my dance group below:
+
+
+![](https://lh6.googleusercontent.com/Le1KzzWrcAxl8sJhrsjEYNRjL1XnN7M4M3LEtpfOk8TOi2UpQ-HC9Flz7TT4GXx8aSrdoec=w16383)
```

Comment:
The link works perfectly! It was a great idea to include it and see in more detail why you love dancing!